nservicebus/recoverability/index.md

```diff
@@ -39,6 +39,8 @@ By default up to five immediate retries are performed if the message processing
 
 Note: The configured value describes the minimum number of times a message will be retried if its processing consistently fails. Especially in environments with competing consumers on the same queue, there is an increased chance of retrying a failing message more times across the endpoints.
 
+Note: Depending on the concurrency and transactionallity settings of the endpoint immediate retries might happen even if configured to be off. To ensure that no retries is performed the endpoints needs to be configured to be either [non transactional](/nservicebus/transports/transactions.md) or the [concurrency retricted to 1](/nservicebus/operations/tuning.md).
```

Comment:
So in essence changing the note to: "The accuracy of the immediate retries depends varies per transport, see the caveats section of the relevant transport for more details"

And then add a caveats section to MSMQ +  sql describing how immediate retries isn't accurate when scaled out + running with concurrency > 1 (cc @Particular/sqlserver-transport-maintainers )

@Particular/docs-maintainers @Particular/nservicebus-maintainers does this make sense?